packages/SwingSet/misc-tools/parse-keys2.py

```diff
@@ -10,7 +10,7 @@ def load_genesis_keys(fn):
         monikers[hexaddr] = moniker
 load_genesis_keys("/home/warner/.ag-chain-cosmos/config/genesis.json")
 
-cmd = ["ag-cosmos-helper", "query", "tendermint-validator-set"]
+cmd = ["agd", "query", "tendermint-validator-set"]
```

Comment:
This made me wonder if the name `agd` might conflict with other stuff in `$PATH`. `apt-cache search agd` shows nothing relevant.

A google search turns up some dental academy but nothing likely to collide.